src/components/FileList/FileList.js

```diff
@@ -124,6 +124,12 @@ export default class FileList extends Component {
     if (!row || !definition) return null
     const { provider, namespace, name, revision } = definition.coordinates
     const path = get(row, 'path')
+    if (row.token)
+      return (
+        <a href={`https://api.clearlydefined.io/attachments/${row.token}`} target="_blank" rel="noopener noreferrer">
```

Comment:
Is there a way to use the current environment prefix the same way we use the definitions API?

Localhost/dev-api/api

Reply:
Sure, we can definitely use the environment var